pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -137,6 +142,11 @@ stdenv.mkDerivation ({
 
   postPatch = ''
     rm -rf obj-x86_64-pc-linux-gnu
+
+    # needed for enabling webrtc+pipewire
+    substituteInPlace \
+      media/webrtc/trunk/webrtc/modules/desktop_capture/desktop_capture_generic_gn/moz.build \
+      --replace /usr/include ${pipewire.dev}/include
```

Comment:
Do other libraries need their include dirs listed here?

Reply:
Why would they?

Reply:
Looking at the patch we should probably also patch at least one other library:
```
+    CXXFLAGS += [ "-I/usr/include/pipewire-0.3" ]
+    CXXFLAGS += [ "-I/usr/include/spa-0.2" ]
```

Reply:
Ok, these two both come from `pipewire` to this patching is actually correct.